src/foam/u2/CheckBox.js

```diff
@@ -75,7 +75,7 @@ foam.CLASS({
 
   css: `
     ^ {
-      margin: 4px;
+      margin: 4px 0px;
```

Comment:
```suggestion
      margin: 0;
```

Reply:
I tried that but the checkbox becomes too close to the label

Reply:
That's not what I'm seeing. With no margin, it's exactly the same distance from the label as text inputs, whereas with 4px it's (obviously) 4px lower than it should be.

<img width="479" alt="Screen Shot 2019-10-04 at 12 48 45 PM" src="https://user-images.githubusercontent.com/4259165/66224990-531e5580-e6a5-11e9-8cc8-9a7606102088.png">

Reply:
You're right, in my pc (chrome or safari) there is absolutely no space for both input and checkboxes, so you're right it's more consistent to set all margins to 0


<img width="535" alt="Screen Shot 2019-10-04 at 10 37 10 AM" src="https://user-images.githubusercontent.com/7996204/66225559-b361c700-e6a6-11e9-9d2f-1370e393b968.png">